docs/build/tutorials/mev-auction-sdk-guide.md

```diff
@@ -0,0 +1,339 @@
+# Kaia MEV Auction SDK Guide for Searchers
+
+[Kaia v2.1.0](https://github.com/kaiachain/kaia/releases/tag/v2.1.0) introduced the MEV Auction system, enabling searchers to participate in fair, transparent auctions for MEV opportunities. This guide provides a comprehensive walkthrough of the searcher workflow using the Kaia MEV Auction SDK.
+
+:::info
+
+This guide uses Kairos testnet endpoints and contract addresses. The mainnet launch is scheduled at the beginning of December 2025. When deploying to mainnet, update all endpoints and contract addresses accordingly.
+
+:::
+
+The searcher workflow consists of four main steps:
+
+![](/img/build/tutorials/searcher-guide-1.png)
+
+1. **Deposit**: Searchers deposit KAIA tokens into the `AuctionDepositVault` to fund bidding activity
+2. **Bid**: Searchers compete by submitting sealed bids to the Auctioneer for backrun slots
+3. **Submit Winning Bid**: The Auctioneer selects winners and forwards winning bids to Consensus Nodes (CNs)
+4. **Execute Bid Transaction**: CNs execute the winning bid transactions through the `AuctionEntryPoint` contract
+
+For detailed technical background, see [KIP-249](https://kips.kaia.io/KIPs/kip-249).
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+- A funded wallet with KAIA tokens for deposits
+- [Go](https://golang.org/) installed (version 1.21+) for SDK examples
+- Access to Kaia network endpoints (this guide uses Kairos testnet)
+- (Optional) [Foundry](https://getfoundry.sh/) installed (for `cast` commands)
+
+**Network Endpoints:**
+
+- Kairos (testnet): `https://public-en-kairos.node.kaia.io`
+- Mainnet: `https://public-en.node.kaia.io`
+
+**Contract Addresses (Kairos):**
+
+- AuctionFeeVault: `0xE4e7d880786c53b6EA6cfA848Eb3a05eE97b2aCC`
+- AuctionDepositVault: `0x2A168bCdeB9006eC6E71f44B7686c9a9863C1FBc`
+- AuctionEntryPoint: `0x2fF66A8b9f133ca4774bEAd723b8a92fA1e28480`
+
+For mainnet contract addresses (available after mainnet launch), check [Contract Addresses](../../references/contract-addresses.md).
+
+:::tip[Monitor MEV Opportunities]
+
+Searchers can identify profitable transactions by:
+
+- **Subscribing to the Auctioneer's pending transaction API**: This API streams transactions directly from Consensus Nodes, allowing you to detect MEV opportunities in real-time. See the [Subscribe Pending Transactions](#step-3-subscribe-to-pending-transactions) section below.
+- **Monitoring the network mempool independently**: Implement your own MEV opportunity detection logic by connecting directly to Kaia nodes.
```

Comment:
Good at overall. Rather, `by connection directly to Kaia nodes`, `by subscribing pending tx` would be more appropriate.